### PR TITLE
Resolves error handler page

### PR DESCRIPTION
**Pull request by monotera**

### CODE REVIEW

File: src/routes/index.js [hunk 1]
```diff
@@ -2,16 +2,26 @@ const express = require('express');
 const router = express.Router();
 const Products = require('../models/Products');
 
-router.get('/', async (req,res) => {
+router.get('/', async (req, res) => {
     try {
-        const prds = await Products.find();
+        const products = await Products.find();
+        if (!products) return res.status(404).render("pages/error", {
+            title: "Moramia <3",
```
Comment (pushdev-code) [inline src/routes/index.js:9]: 404 title should be different. Try something like: `Page not found`

File: src/routes/index.js [hunk 2]
```diff
@@ -2,16 +2,26 @@ const express = require('express');
...
         res.render("pages/index", {
             title: "Moramia <3",
-            products : prds,
+            products: products,
+            refreshUrl: process.env.BROWSER_REFRESH_URL, //browser refresh listening
+        });
+    } catch (err) {
+        console.log(err);
+        res.status(500).render("pages/error", {
+            title: "Moramia <3",
```
Comment (pushdev-code) [inline src/routes/index.js:21]: same for 500 title: Try something like: `Internal server error`

File: src/static/css/error.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    background-clip: text;
+    background: rgb(34, 193, 195);
+    background: linear-gradient(45deg, rgba(34, 193, 195, 1) 35%, rgba(253, 45, 45, 1) 85%);
+    color: transparent;
+    font-family: $font-stack;
+    font-size: 15rem;
+    font-weight: 200;
+    margin: 0;
+    text-transform: uppercase;
+    -webkit-background-clip: text;
```
[the remark this answers is not in the document]
Comment (monotera) [reply src/static/css/error.scss:23]: without this property it doesn't work

File: src/static/css/error.scss [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+      border: 1px solid #efefef;
+      border-radius: 50%;
+      color: $moramia-blue;
+      display: inline-block;
+      font-size: 14px;
+      height: 40px;
+      line-height: 40px;
+      margin: 3px;
+      transition: 0.2s all;
+      -webkit-transition: 0.2s all;
```
[the remark this answers is not in the document]
Comment (monotera) [reply src/static/css/error.scss:81]: Done

File: src/static/css/error.scss [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+      @media screen and (min-width: $breakpoint-tablet) {
+        display: none;
+      }
+    }
+
+    :hover {
+      background-color: $moramia-blue;
+      border-color: $moramia-blue;
+      color: #fff;
```
[the remark this answers is not in the document]
Comment (monotera) [reply src/static/css/error.scss:92]: Done

File: src/static/css/error.scss [hunk 6]
```diff
@@ -0,0 +1,95 @@
+.error-section {
+  margin: auto;
+  width: 100%;
+
+  @media screen and (max-width: $breakpoint-tablet) {
```
[the remark this answers is not in the document]
Comment (monotera) [reply src/static/css/error.scss:5]: Done

File: src/routes/index.js (hunk 1 above)
Comment (monotera) [reply src/routes/index.js:9]: Done

File: src/routes/index.js (hunk 2 above)
Comment (monotera) [reply src/routes/index.js:21]: Done

File: src/views/partials/error.ejs [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    <div class="notfound">
+        <div class="notfound-404">
+            <h1 class="notfound-h1"><%= error %></h1>
+        </div>
+        <h2 class="notfound-h2">Oops! Nothing was found</h2>
+        <p class="notfound-p">
+            The page you are looking for might have been removed,
+            had its name changed or is
+            temporarily unavailable. <a class="notfound-return" href="/">Return to homepage</a></p>
+        <div class="notfound-social">
```
[the remark this answers is not in the document]
Comment (monotera) [reply src/views/partials/error.ejs:11]: Done

File: src/static/css/error.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (pushdev-code) [reply src/static/css/error.scss:23]: ok. we need to take a look at our prefixer. it is ok for now.

**Pull request merged**